### PR TITLE
Github OAuth CLI Login

### PR DESCRIPTION
**Pull request by depombo**
Screen Shot 2021-03-25 at 12 46 29 PM
Screen Shot 2021-03-25 at 12 48 06 PM
Screen Shot 2021-03-25 at 12 51 18 PM
Screen Shot 2021-03-25 at 12 51 23 PM

### CODE REVIEW

**Comment by dfellis** [comment]
How do we make the icon bigger? ;)

**Comment by depombo** [comment]
> How do we make the icon bigger? ;)

I know! I tried, but we can't. We can only pick the background color :/

**Review by aguillenv — APPROVED** [review]

File: cli/src/deploy/mod.rs [hunk 1]
```diff
@@ -22,6 +23,9 @@ const FORBIDDEN_OPERATION: &str =
   "Please review your credentials. Make sure you have follow all the \
   configuration steps: https://alantech.gitbook.io/anycloud/";
 const NAME_CONFLICT: &str = "Another application with same app ID already exists.";
+const UNAUTHORIZED_OPERATION: &str =
+  "Invalid AnyCloud authentication credentials. Please try again and you will be asked \
+  to login once again.";
```
Comment (dfellis) [inline cli/src/deploy/mod.rs:28]: A little awkward to have "again" twice in a sentence like that. Maybe "Please retry and you will be asked to reauthenticate." ?

**Review by dfellis — APPROVED** [review]
Only one minor comment

File: cli/src/deploy/mod.rs (hunk 1 above)
Comment (depombo) [reply cli/src/deploy/mod.rs:28]: yes!

**Pull request merged**